PhysicsTools/NanoAOD/plugins/BuildFile.xml

```diff
@@ -17,6 +17,12 @@
 <use   name="DQMServices/Core"/>
 <use   name="CondFormats/BTauObjects"/>
 <use   name="CondTools/BTau"/>
+<use   name="DataFormats/CTPPSDetId"/>
+<use   name="DataFormats/CTPPSReco"/>
+<use   name="DataFormats/ProtonReco"/>
+<use   name="CondFormats/RunInfo"/>
+<use   name="CondFormats/DataRecord"/>
+<use   name="RecoCTPPS/ProtonReconstruction"/>
```

Comment:
remove <use name="RecoPPS/ProtonReconstruction"/>
as done in master
https://github.com/cms-sw/cmssw/commit/8221dfb5c7f9892fc0646a6bb39e40b84c2a0368#diff-98594af19159a28fc4fe8d293e61a4cc09398551b79f2b2d066551ccf6cf11e5

Reply:
Done.